packages/components/src/UserStatement/RequestAppealStatement.tsx

```diff
@@ -178,6 +177,13 @@ export class RequestAppealStatement extends React.Component<RequestAppealStateme
     return !summaryValue || !summaryValue.length || !citeConstitution.innerText.length || !details.innerText.length;
   };
 
+  private renderHelperMessage = (): JSX.Element => {
+    if (this.isFormInvalid()) {
```

Comment:
does this actually block the user from moving forward? I think it'd be good if the button was disabled EITHER if the field is empty OR invalid (but empty wouldn't have error message?)

Reply:
The only issue is since the fields are required, if a user leaves a field empty, there is no messaging that informs the user that they're missing a required field (other than a tiny bit of text by the input field).

Maybe a better solution would be to never disable the button and add the validation as a preTransactions method... or add support for a separate validator fn prop for the Transaction button, which would then trigger the display of messaging.